luacompiler/src/lib/bytecode/instructions.rs

```diff
@@ -45,6 +45,8 @@ pub fn format_instr(instr: u32) -> String {
         15 => "Push",
         16 => "VarArg",
         17 => "Eq",
+        18 => "MovR",
```

Comment:
Why MovR and not just Mov?

Reply:
I could technically "merge" `mov` with `movr`, but then the bytecode would become a bit confusing, but I might be wrong! I could, for instance, treat `mov 1, 0, 0` as the usual `mov` instruction, and `mov 1, 0, 1` as `movr`.

Reply:
What's the difference between the two?

Reply:
What's the third argument in each case?

Reply:
In the case of `mov` it is ignored. In the case of `movr`, if the last argument is `1` or `2`, then all return values are pushed to the stack.

Reply:
These cases are documented [here](https://github.com/rbartlensky/Lua-interpreter/pull/21/files#diff-230b6f762494d733b568f0f309625819R108).

Reply:
OK.